.github/workflows/lint.yml

```diff
@@ -27,3 +27,25 @@ jobs:
         shell: bash
       - run: go version
       - run: make lint
+
+  prettify:
+    name: Prettify
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

Comment:
Why is this needed? The CI runs on the pushed code or PR already AFAIK.

Reply:
You mean why used checkout action? 
Initially, I thought to just amend the commit and not create a new one, so I pulled out all the brunches and stories. But with a pull request, there was no reason to pull everything, so the checkout is only for the head. 
So at the push event, I left it as it was with the job on the linter above, if you look, there is also a checkout

Reply:
I mean why is `if: github.event_name == 'pull_request'` vs. `if: github.event_name == 'push'` needed. If you combine the two you will get the "current code" (on a PR it will include the PRed code, on main it will include HEAD).

Reply:
Yes, I think I understand your train of thought. Well, I will commit without separation, with a depth of 1

```
- uses: actions/checkout@v3
  with: { fetch-depth: 1 }
```